core/src/main/scala/org/elasticmq/actor/queue/QueueActorMessageOps.scala

```diff
@@ -1,211 +1,40 @@
 package org.elasticmq.actor.queue
 
-import akka.actor.ActorRef
-import org.elasticmq.actor.queue.ReceiveRequestAttemptCache.ReceiveFailure.{Expired, Invalid}
+import akka.actor.Timers
+import org.elasticmq.actor.queue.operations._
 import org.elasticmq.actor.reply._
 import org.elasticmq.msg.{DeleteMessage, LookupMessage, ReceiveMessages, SendMessage, UpdateVisibilityTimeout, _}
 import org.elasticmq.util.{Logging, NowProvider}
-import org.elasticmq.{MessageData, MessageId, MillisNextDelivery, NewMessageData, _}
 
-trait QueueActorMessageOps extends Logging {
+import scala.concurrent.duration.DurationInt
+
+trait QueueActorMessageOps
+    extends Logging
+    with SendMessageOp
+    with UpdateVisibilityTimeoutOps
+    with DeleteMessageOps
+    with ReceiveMessageOps
+    with MoveMessageOps
+    with Timers {
   this: QueueActorStorage =>
 
   def nowProvider: NowProvider
   def context: akka.actor.ActorContext
 
+  timers.startTimerWithFixedDelay(s"Timer: ${queueData.name}", DeduplicationIdsCleanup, 1.second)
```

Comment:
shouldn't this be done in actor's `postStart` or `preStart` or some callback like that?

Reply:
It is already done in the `Timers` trait